frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -246,16 +246,16 @@ function roleSelected(evt: any) {
 
                     <div class="form-group">
                         <div class="row mb-0">
-                            <label for="userIdInput" class="control-label px-0"
+                            <span id="userIdInput" class="control-label px-0"
                                 >Type user's domain name
-                                <span class="text-danger"> *</span></label
+                                <span class="text-danger"> *</span></span
                             >
                         </div>
 
                         <div class="row mt-0">
                             <div class="col-md-3 px-0">
                                 <Field
-                                    id="userIdInput"
+                                    aria-labelledby="userIdInput"
```

Comment:
As you remove the "id" from <Field>, does this input needs an "id"?
Should there be an "id" (or similar id attribute from PrimeVue) at line 265 "\<InputText>"?

I am not sure the original issue was to do with using "**aria-labelledby**", maybe it was just complaining about previous "\<label>" field (with for=) does not find a matching "id" input field. So perhaps the id was missing in line 265 is the cause?

Reply:
the primevue give us the following accessibility options: via label tag combined with id or using aria-labelledby, aria-label. The label tag combined with id show the error which was described  on the task

Reply:
Are you sure it is the **("label tag combined with id show the error")**? My understanding is combination "<label for=>" with form field's "id" are basic html usage. So with the original errors and warnings on console the cause seems to be that the "id" is placed at the wrong place, since <Field> isn't a normal html/vue component, but a validation library component, so probably it does not render id.
Could you try to verify this? Thank you.

Reply:
Catherine already pushed some fixes for this.
That's basically the idea for <label> and input field id match I was getting you to.
Resolve this comment.